src/EnergyPlus/CMakeLists.txt

```diff
@@ -612,3 +612,49 @@ if(UNIX AND NOT APPLE)
   COMMAND ${CMAKE_COMMAND} -E create_symlink energyplus EnergyPlus
   COMMAND ${CMAKE_COMMAND} -E rename EnergyPlus "${CMAKE_BINARY_DIR}/Products/EnergyPlus") 
 endif()
+
+# if we are building a package, we can do additional operations, such as run doc scripts
+if(BUILD_PACKAGE)
+  
+  # Some docs are generated on the fly here, create a dir for the 'built' files
+  set( DOCS_OUT "${CMAKE_BINARY_DIR}/autodocs" )
+  file( MAKE_DIRECTORY "${DOCS_OUT}" )
+  
+  # the output variables listing
+  add_custom_command( 
+    TARGET energyplus
+    PRE_BUILD
+    COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/doc/tools/parse_output_variables.py" "${CMAKE_SOURCE_DIR}/src/EnergyPlus" "${DOCS_OUT}/SetupOutputVariables.csv" "${DOCS_OUT}/SetupOutputVariables.md" 
+  )
+  install(FILES "${CMAKE_BINARY_DIR}/autodocs/SetupOutputVariables.csv" DESTINATION "./")
+  install(FILES "${CMAKE_BINARY_DIR}/autodocs/SetupOutputVariables.md" DESTINATION "./")
+  
+  # the example file summary
+  add_custom_command(
+    TARGET energyplus
+    PRE_BUILD
+    COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/doc/tools/example_file_summary.py" "${CMAKE_SOURCE_DIR}/testfiles" "${DOCS_OUT}/ExampleFiles.html"
+  )
+  install(FILES "${DOCS_OUT}/ExampleFiles.html" DESTINATION "./ExampleFiles/")
+  
+  # the example file objects link
+  add_custom_command(
+    TARGET energyplus
+    PRE_BUILD
+    COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/doc/tools/example_file_objects.py" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Energy+.idd" "${CMAKE_SOURCE_DIR}/testfiles" "${DOCS_OUT}/ExampleFiles-ObjectsLink.html"
+  )
+  install(FILES "${DOCS_OUT}/ExampleFiles-ObjectsLink.html" DESTINATION "./ExampleFiles/")
+  
+  # the change log, only if we do have a github token in the environment
+  if(NOT "$ENV{GITHUB_TOKEN}" STREQUAL "")
```

Comment:
The github token _is_ being passed in via decent_ci.  We were already passing it in to decent_ci, and it just passes it along into the environment.